Assets/Thirdweb/Scripts/Bridge.cs

```diff
@@ -90,17 +98,15 @@ public static async Task<T> InvokeRoute<T>(string route, string[] body)
             var task = new TaskCompletionSource<string>();
             taskMap[taskId] = task;
             ThirdwebInvoke(taskId, route, msg, jsCallback);
-            try 
-            {
-                string result = await task.Task;
-                // Debug.LogFormat("Result from {0}: {1}", route, result);
-                return JsonConvert.DeserializeObject<Result<T>>(result).result;
-            } 
-            catch (Exception)
+            string result = await task.Task;
+            Debug.Log($"InvokeRoute Result: {result}");
+            if (typeof(T) == typeof(string))
```

Comment:
so it only returns the error if you explicitely ask for a string? when you invoke? That's doesn't seem great.

I have a better idea for this. The encapsulating Result type will have 2 implementations, one for success which boxes the actual result of type T (like now) and one for error which will contain the exception and possibly more fields like a `reason` property.

I'm going to work on this this week, will probably take over this PR

Reply:
Definitely needs a rework, but this is the only workaround that worked for me to not get parsing issues with the current Result type from thirdweb.js when returning a single string that has unparseable stuff in it like dots, really a pain if this isn't done OR the actual result could be serialized once again on the js side I guess?